### ISSUE TITLE
Use our standard button element on the Create Account screen

### ISSUE THREAD

**Issue by luixxiul** [vector-im/element-web#25037]
On the Create Account screen, the "Accept" button is a plain HTML button instead of our standard `AccessibleButton` class.

This could cause it to behave inconistently with other buttons in our product, and it may be missing some accessibility features.

The relevant code is: `InteractiveAuthEntryComponents.tsx` for https://github.com/matrix-org/matrix-react-sdk/blob/e5a314617ab9a18fbf773768933c4e52b845b2fd/src/components/views/auth/InteractiveAuthEntryComponents.tsx#L388-L399

`submitButton` should be replaced with `AccessibleButton`.

**Comment by andybalaam** [comment]
@luixxiul I updated the title and description to explain this problem from a more user-centric viewpoint. Please do correct it if I misunderstood.

**Comment by luixxiul** [comment]
Thanks, that is exactly what I wanted to express :-D
